frontend/src/static/js/v_authorship.js

```diff
@@ -298,6 +312,18 @@ window.vAuthorship = {
       window.encodeHash();
     },
 
+    updateSelectedFiles() {
+      this.$store.commit('updateIsLoadingOverlayEnabled', true);
+      const selectedFiles = this.files.filter(
```

Comment:
Shouldn't we put the whole computation inside `setTimeOut` so that the loading overlay will show up before the actual computation?

Reply:
Done